templates/editSchedule.html

```diff
@@ -64,6 +70,10 @@
   </div>
 </template>
 
+<template name="unscheduledRound">
+  <div class="fc-event" data-round-id="{{ _id }}" data-duration="01:00">{{ title }}</div>
```

Comment:
Could we consolidate this `data-duration` attribute with `DEFAULT_ROUND_DURATION_MINUTES` defined at the top of client/editSchedule.js?